manifests/repo.pp

```diff
@@ -4,8 +4,8 @@
 #
 #   include homebrew::repo
 class homebrew::repo (
-  $installdir   = $homebrew::config::installdir,
-  $min_revision = $homebrew::config::min_revision,
+  $installdir   = $homebrew::installdir,
+  $min_revision = $homebrew::min_revision,
```

Comment:
hmm... I thought the class ``homebrew::repo`` should be using the same configurations as the main class ``homebrew``.

But if it was intentional, i would remove it